scripts/build-resolver-cache.js

```diff
@@ -0,0 +1,52 @@
+const fs = require("fs");
+const path = require("path");
+const shell = require("shelljs");
+const getMethodsForTest = require("../tests/getMethodsForTest");
+
+let focusedMethods = require("../tests/focusedMethods");
+const methodsForTest = getMethodsForTest(focusedMethods);
+
+const unresolveable = [];
+
+// Run locally before push to decrease build time.
```

Comment:
nit: maybe be a bit more clearer that this script is not actually run in the CI. I was a bit concerned wrt an apparent dependency on uniresolver.io in CI.

Reply:
I have added a comment to readme to make this clearer.